pkgs/applications/misc/waypaper/default.nix

```diff
@@ -38,13 +37,14 @@ python3.pkgs.buildPythonApplication rec {
   meta = with lib; {
     changelog = "https://github.com/anufrievroman/waypaper/releases/tag/${version}";
     description = "GUI wallpaper setter for Wayland-based window managers";
+    homepage = "https://github.com/anufrievroman/waypaper";
+    license = licenses.gpl3Only;
```

Comment:
Undo this. By unwritten convention, short `description` and `longDescription` stand next each other. 

```suggestion
```